Makefile.include

```diff
@@ -123,6 +123,15 @@ UNAME := $(shell uname -m -s)
 OS = $(word 1, $(UNAME))
 OS_ARCH = $(word 2, $(UNAME))
 
+# generate a canary value for the ssp module
```

Comment:
As this is currently only used in sys/ssp, could we move also move this into `sys/ssp/Makefile`?

Reply:
I attempted to do this in `sys/ssp/Makefile.include`. Unfortunately, this does not seem to work as the module specific `Makefile.include` is included repeatedly which results in a different canary on each include. I explained this further in the commit message of 23cb94b7e342c45512a611166bf552a3a9f3fe8b.

Reply:
> Currently testing whether adding it to sys/ssp/Makefile work though...

Does not seem to work see https://github.com/RIOT-OS/RIOT/pull/13119#discussion_r366471816